app.py

```diff
@@ -1,41 +1,78 @@
-from flask import Flask,request
+from flask import Flask, request
 from flask.json import jsonify
 
 app = Flask(__name__)
 
+
 class Availability:
     def __init__(self, start, end):
         self.start = start
         self.end = end
+
     def to_json(self):
         return {"start": self.start, "end": self.end}
 
-class Teacher:
-    def __init__(self, name, location, availabilities, price):
-        self.name = name
+
+class User:
```

Comment:
Let's not add baseclass for User yet. There is a powerful rule in programming: YAGNI.
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it